contracts/Bulker.sol

```diff
@@ -18,15 +20,18 @@ contract Bulker {
     uint public constant ACTION_TRANSFER_ASSET = 3;
     uint public constant ACTION_WITHDRAW_ASSET = 4;
     uint public constant ACTION_WITHDRAW_ETH = 5;
+    uint public constant ACTION_CLAIM_REWARD = 6;
 
     /** Custom errors **/
     error InvalidArgument();
     error FailedToSendEther();
     error Unauthorized();
 
-    constructor(address admin_, address comet_, address payable weth_) {
+    // XXX do we want to be constantizing the comet address and rewards address
```

Comment:
This question came up while working on this with 🥈. Do we want to make `Bulker` more generic by no longer making `comet` (and `rewards`) an immutable and instead, have it be passed in as calldata during `invoke`? This would allow us to use a single `Bulker` for multiple deployments on one chain. It also means we wouldn't have to re-deploy `Bulker` whenever the `rewards` contract changes.

Reply:
Sounds like a good idea, redeploying for multiple instances would suck, especially if we had different versions of bulker

Reply:
Cool, I'll get rid of the XXX and start a new branch for this change.

Reply:
Created #542 for this